spec/javascripts/tests/Header_spec.js

```diff
@@ -0,0 +1,65 @@
+describe('Header', function() {
+  'use strict';
+
+  beforeEach(function(done) {
+    var self = this;
+
+    fixture.setBase('spec/javascripts/fixtures');
+
+    requirejs(
+      ['jquery', 'Header'],
+      function($, Header) {
+        fixture.load('Header.html');
+
+        self.$header = $(fixture.el).find('[data-dough-component="Header"]');
+        self.obj = new Header(self.$header);
+
+
+        self.activeClass = 'mobile-header__search-active';
+        self.$header = $(document).find('[data-site-header]');
+        self.$mobileSearchButton = self.$header.find('[data-mobile-search-button]');
+
+        var $openText = $(fixture.el).find('[data-mobile-search-text]');
+        self.expected = {
+          openText: "Open Search",
+          closeText: "Close Search",
```

Comment:
As above, can we access these values from Header.js?
openText: self.obj.$mobileSearchText.html()
closeText: self.obj.$mobileSearchText.data('mobile-search-text-close')

Reply:
fc70ddd
c951acf

Reply:
config change b87229d breaks tests, removed all tests 60ba64d